README.md

```diff
@@ -1 +1,16 @@
 # heiten-app
+
+## Getting Started
```

Comment:
You can fill these in later like after we have everything up and running, just wanted to scaffold them out so you can have them on the back of your mind. This is huge b/c this is the first think a potential employer will see when you share your github link so you want it to be legit. I'll find some examples for you to look at that might be helpful as well.